ios/RNSScreenStack.mm

```diff
@@ -61,6 +61,21 @@ - (UIViewController *)childViewControllerForStatusBarHidden
   return [self topViewController];
 }
 
+- (void)viewDidLayoutSubviews
+{
+  [super viewDidLayoutSubviews];
+  if ([self.topViewController isKindOfClass:[RNSScreen class]]) {
+    RNSScreen *screenController = (RNSScreen *)self.topViewController;
+
+    // Calculate header height during simple transition from one screen to another.
+    // If RNSScreen includes a navigation controller of type RNSNavigationController, it should not calculate
+    // header height, as it could have nested stack.
+    if (!screenController.hasNestedStack) {
```

Comment:
`hasNestedStack` is a method, not property. Does this call actually work?

Reply:
As I investigated this method it looks like this method is being called right now (which is strange, indeed). It looks that even without this method it reports correct values, but this method is also reducing event calls to JS, so I would prefer to leave it here 🤞